test/scripts/deploy_with_helm.sh

```diff
@@ -156,6 +156,8 @@ if [ -z "$ONLY_DB" ]; then
   chmod og-rwx ~/.flightctl/certs/*.key
 fi
 
+./bin/flightctl login --insecure-skip-tls-verify https://api.${IP}.nip.io:3443
```

Comment:
# attempt to login, it could take some time for API to be stable
```suggestion
for i in {1..5}; do
  if ./bin/flightctl login --insecure-skip-tls-verify https://api.${IP}.nip.io:3443; then
    break
  fi
  sleep 5
done
```

Locally this worked better, somehow my API needed a few more seconds to be fully functional and responding.